### PR TITLE
add CuVoodoo org, request ID for CuVoodoo bootloader and BusVoodoo adapter

### PR DESCRIPTION
**Pull request by cuvoodoo**
Add CuVoodoo organisation, request USB ID for CuVoodoo USB DFU bootloader (firmware only) and BusVoodoo multi-protocol debugging adapter (hardware and firmware).

### CODE REVIEW

**Comment by Arachnid** [comment]
There's no indication of license in the source code repository. Could you add a license/copying file, please?

**Comment by cuvoodoo** [comment]
Issue fixed: I've added LICENSE.txt files in both firmware/software repositories.
The source files also mention the license at the top.

**Comment by cuvoodoo** [comment]
The CI check failed, but I couldn't see in the CI platform why.
Looking at the CI history the two previous builds also failed. Maybe there is a issue independent of this PR. If this is not the case, could you tell how the build is done or where I overlooked build information?

File: 1209/4256/index.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: BusVoodoo multi-protocol debugging adapter
+owner: CuVoodoo
+license: CERN OHL v.1.2, GNU GPLv3
+site: https://bus.cuvoodoo.info/
+source: https://bus.cuvoodoo.info/
```
Comment (Arachnid) [inline 1209/4256/index.md]: Can you please link to the source code here?

**Comment by cuvoodoo** [comment]
Issue fixed: I set the source link directly to the git repositories

File: 1209/4356/index.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: CuVoodoo firmware
+owner: CuVoodoo
+license: GNU GPLv3
+site: https://wiki.cuvoodoo.info/doku.php?id=stm32f1xx#usb_dfu
+source: https://git.cuvoodoo.info/stm32f1/
+--
```
Comment (Arachnid) [inline 1209/4356/index.md]: You are missing a hyphen here to terminate the metadata block.

**Comment by cuvoodoo** [comment]
Thanks for the review. Issue is now fixed.
Would if be possible to list all my mistakes at once so you don't have to review it so often and I don't have to wait two to three weeks between reviews?

**Comment by Arachnid** [comment]
I would, but you [introduced that typo fixing a previous issue](https://github.com/pidcodes/pidcodes.github.com/pull/296/commits/d9b2f62eefcddf10cfff7c4492380d7f5c2fa70e).

**Pull request merged**